cards.html

```diff
@@ -4,9 +4,121 @@
+            </div>
+        <div class="heading">
+            <h4><strong>Puma</strong>
+            <span class="btn btn-icon-primary wishlist-btn"><i class="far fa-heart"></i></span>
+            </h4>
+            <h5>White men regular cotton t-shirt.</h5>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
+        <a class="btn btn-outline-primary"><span><i class="fas fa-shopping-cart"></i></span></a>
+        </div>
+    </section>
+    <!-- CARD CONTAINER-3 -->
+    <section class="card-container">
+        <div class="content">
+        <img class="image" src="assets/shirt.jpg" alt="img"/>
+        <div class="heading-overlay">
+            <h4 class="text-overlay">
+            <strong>Puma</strong>
+            </h4>
+            <h5 class="product-info">White men regular cotton t-shirt.</h5>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
+        <a class="btn btn-outline-primary"><span><i class="fas fa-shopping-cart"></i></span></a>
+        </div>
+    </section>
+    <!-- CARD CONTAINER-4 -->
+    <section class="card-container">
+        <div class="content">
+            <div class="image">
+                <img src="assets/shirt.jpg" alt="img"/>
+                <span class="close-btn"><i class="fas fa-times"></i></span>
+            </div>
+
+        <div class="heading">
+            <h4>
+            <strong>Puma</strong>
+            <span class="btn btn-icon-primary wishlist-btn"><i class="far fa-heart"></i></span>
+            <h5>White men regular cotton t-shirt.</h5>
+            </h4>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
+        <a class="btn btn-outline-primary"><span><i class="fas fa-shopping-cart"></i></span></a>
+        </div>
+    </section>
+    <!-- CARD CONTAINER-5 -->
+    <section class="card-container">
+        <div class="content">
+        <!-- <img class="image" src="assets/shirt.jpg" alt="img"/> -->
+        <div class="heading">
+            <h4>
+            <strong>Puma</strong>
+            <span class="btn btn-icon-primary wishlist-btn"><i class="far fa-heart"></i>
+            </span>
+            </h4>
+            <h5>White men regular cotton t-shirt.</h5>
+        </div>
+        <h4 class="item-cost"><strong><i class="fas fa-rupee-sign"></i> 1,899</strong></h4>
+        <div>
+        <a><button class="btn btn-solid-primary">Buy Now</button></a>
+        <a class="btn btn-outline-primary"><span><i class="fas fa-shopping-cart"></i></span></a>
```

Comment:
I think this span tag can be omitted if it does not have a class attribute.

Reply:
Done